### DISCUSSION TITLE
Campus

### DISCUSSION THREAD

**Discussion by pulgamecanica** [pulgamecanica/42Portfolio#66] [Ideas]
Should show campus?

What about people who have more than one?

- Show active campus

**Comment by PukieDiederik** [comment]
I think it's good to show the current campus of a person. Although the curriculum should be the same across campuses, I do believe the culture within each campus can be different. This could also be a toggle-able feature, so people who don't want to show their campus can hide it.

I don't know if we will have information available for previous campuses a user was in. If we can retrieve this information I think it will only be valuable information if we can also supply the dates whenever a user was active at a campus. And if a user only wants to show their active campus we can also make that a toggle-able setting.
